recipes/tomcat.rb

```diff
@@ -22,6 +22,14 @@
       config(setenv_options)
     end
   end
+
+  apache_tomcat_config 'server' do
+    options do
```

Comment:
if you don't set the template name/cookbook is not by default taking the chef-apache_tomcat one?